src/Core/Resolvers/MsSqlQueryExecutor.cs

```diff
@@ -217,9 +217,9 @@ public override string GetSessionParamsQuery(HttpContext? httpContext, IDictiona
             foreach ((string claimType, string claimValue) in sessionParams)
             {
                 string paramName = $"{SESSION_PARAM_NAME}{counter.Next()}";
-                parameters.Add(paramName, new(claimValue));
+                parameters.Add(paramName, new(claimValue));          
```

Comment:
```suggestion
                parameters.Add(paramName, new(claimValue));
```
nit: removing trailing spaces